flink-libraries/flink-table/src/test/scala/org/apache/flink/api/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -1101,6 +1101,45 @@ class ScalarFunctionsTest extends ExpressionTestBase {
       "true")
   }
 
+  @Test
+  def testInExpressions(): Unit = {
+    testTableApi(
```

Comment:
@twalthr Let's create another Jira for support POJOs in IN operator.
To be honest, I think this is not necessary because we can use something like this:
`SELECT a, b FROM T WHERE a.prop1.IN[1,2,3] AND a.prop2.IN[4,5,6] AND ....`
I don't remember any case when in standard SQL in `IN` we can use something else than primitives and their wrappers.